### ISSUE TITLE
delete default credential

### ISSUE THREAD

**Issue by djfan** [CartoDB/cartoframes#1557]
After `set_default_credentials()`, how can users properly delete this default credential ? 

Anything else other than 
```python
set_default_credentials(username='ghost_account', api_key='default_public')
```
?

**Comment by Jesus89** [comment]
We have just introduced this method: https://github.com/CartoDB/cartoframes/blob/develop/cartoframes/auth/defaults.py#L131. It will be available in 1.0.1.

**Comment by djfan** [comment]
perfect!!
